lib/wisper/registration/object.rb

```diff
@@ -8,7 +8,7 @@ def initialize(listener, options)
       super(listener, options)
       @with   = options[:with]
       @prefix = ValueObjects::Prefix.new options[:prefix]
-      @allowed_classes = Array(options[:scope]).map(&:to_s).to_set
+      @allowed_classes = Array(options[:scope]).to_set
```

Comment:
We should transform `@allowed_classes` right here, because it's only used twice in `publisher_in_scope?`.

```ruby
@allowed_classes = Array(options[:scope])
# Use destructive methods to avoid allocating new arrays.
@allowed_classes.map! { |klass| klass.is_a?(String) ? Object.const_get(klass) : klass }
@allowed_classes.uniq!
```